workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/metrics.py

```diff
@@ -30,7 +30,7 @@ def grab_diag(ds, name):
             replace_dict[var] = var[: -len(match)]
 
     if len(replace_dict) == 0:
-        raise ValueError(f"No diagnostics with name {name} found.")
+        return xr.Dataset()
```

Comment:
in particular since the metrics are now computed for the verification dataset which lacks some diagnostics, it is necessary to make sure the code works for the case of missing diagnostics.